src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -486,150 +512,151 @@ public static class PublishedContentExtensions
+            //                            }
+            //                            else
+            //                            {
+            //                                result = converted;
+            //                            }
+            //                        }
+            //                    }
+            //                    else
+            //                    {
+            //                        // Ensure we pass back an empty enumerable if the expected output is an enumerable.
+            //                        // and null has been returned by the type converter.
+            //                        if (propertyIsEnumerableType)
+            //                        {
+            //                            result = EnumerableInvocations.Empty(typeInfo.GenericTypeArguments.First());
+            //                        }
+            //                    }
+            //                }
+            //                else if (propertyType.IsInstanceOfType(propertyValue))
+            //                {
+            //                    // If the TypeConverter's `CanConvertFrom` has returned false,
+            //                    // then we can check if the value is the same type as the target type.
+            //                    result = propertyValue;
+            //                }
+            //            }
+            //        }
+            //    }
+            //}
+
+            //TODO: Move this to 
+
+            //if (propertyType == typeof(HtmlString))
+            //{
+            //    // Handle Html strings so we don't have to set the attribute.
+            //    var processorType = typeof(HtmlStringProcessor);
+            //    var processor = processorType.GetDependencyResolvedInstance() as DittoProcessor;
+
+            //    if (processor != null)
+            //    {
+            //        // This contains the IPublishedContent and the currently converting property descriptor.
+            //        var descriptor = TypeDescriptor.GetProperties(instance)[propertyInfo.Name];
+            //        var context = new DittoProcessorContext
+            //        {
+            //            Content = content,
+            //            Value = propertyValue,
+            //            PropertyDescriptor = descriptor,
+            //            TargetType = type
+            //        };
+
+            //        // We're deliberately passing null.
+            //        // ReSharper disable once AssignNullToNotNullAttribute
+            //        var processedValue = processor.ProcessValue(context, new DittoProcessorAttribute(typeof(HtmlStringProcessor)), culture);
+            //        if (processedValue is HtmlString)
+            //        {
+            //            result = processedValue;
+            //        }
+            //    }
+            //}
+            //else 
+            if (propertyType.IsInstanceOfType(propertyValue))
```

Comment:
As discussed, these conversion checks could be made as processors and could be added to the end of the list of processors within `.GetProcessedValue`?